lib/rubocop/cop/registry.rb

```diff
@@ -24,34 +24,40 @@ def initialize(name, origin, badges)
     class Registry
       include Enumerable
 
+      attr_reader :options
+
       def initialize(cops = [], options = {})
         @registry = {}
         @departments = {}
         @cops_by_cop_name = Hash.new { |hash, key| hash[key] = [] }
 
-        cops.each { |cop| enlist(cop) }
+        @enrollment_queue = cops
         @options = options
       end
 
       def enlist(cop)
-        @registry[cop.badge] = cop
-        @departments[cop.department] ||= []
-        @departments[cop.department] << cop
-        @cops_by_cop_name[cop.cop_name] << cop
+        @enrollment_queue << cop
+      end
+
+      def delist(cop)
```

Comment:
I think "dismiss" or something along those lines would fit better the overall terminology. :-) Not a native speaker, so I'm not sure if there are better opposites of "enlist" that we can consider.

Reply:
Dismiss is good (not a native speaker either 😸)